core/src/main/java/com/microsoft/applicationinsights/internal/config/ConfigurationFileLocator.java

```diff
@@ -147,8 +147,6 @@ private String getConfigurationAbsolutePath(String path) {
         File configFile = new File(path, configurationFileName);
 
         if (configFile.exists()) {
-            InternalLogger.INSTANCE.logAlways(InternalLogger.LoggingLevel.INFO, "Found configuration file in: '%s'", path);
```

Comment:
I think we should leave this line so in case the user got confused and placed ApplicationInsights.xml in several locations he will be able to tell easily what file was used.